includes/class-alma-wc-payment-gateway.php

```diff
@@ -289,17 +291,6 @@ public function init_form_fields() {
 		}
 	}
 
-	/**
-	 * Return whether or not this gateway still requires setup to function.
-	 *
-	 * @return bool
-	 */
-	public function needs_setup() {
-		$this->update_option( 'enabled', 'yes' );
-
-		return true;
-	}
```

Comment:
👍

Reply:
No trace of needs_setup into parent classes (`WC_Settings_API` and `WC_Payment_Gateway`) which versions are respectively 2.1.0 & 2.6.0 .

I am sure this function is never used because `Alma_WooCommerce_Gateway::update_option` is not declared (`$this->update_option()`)

Reply:
`enabled` => `yes` is defined in `Alma_WC_Settings::default_settings()` default configuration